COCOAPODS.md

```diff
@@ -1,150 +1,381 @@
+`version` is a version of the Gradle project.
+
+    <div class="sample" markdown="1" theme="idea" data-highlight-only>
+    
+    ```kotlin
+    plugins {
+        kotlin("multiplatform") version "{{ site.data.releases.latest.version }}"
+        kotlin("native.cocoapods") version "{{ site.data.releases.latest.version }}"
+    }
+    
+    // CocoaPods requires the podspec to have a version.
+    version = "1.0"
+    
+    kotlin {
+        cocoapods {
+            // Configure fields required by CocoaPods.
+            summary = "Some description for a Kotlin/Native module"
+            homepage = "Link to a Kotlin/Native module homepage"
+    
+            // You can change the name of the produced framework.
+            // By default, it is the name of the Gradle project.
+            frameworkName = "my_framework"
+        }
+    }
+    ```
+    
+    </div>
+    
+5. Re-import the project.
 
 When applied, the CocoaPods plugin does the following:
 
-1. Adds both `debug` and `release` frameworks as output binaries for all iOS and macOS targets.
-2. Creates a `podspec` task which generates a [podspec](https://guides.cocoapods.org/syntax/podspec.html)
-file for the given project.
+* Adds both `debug` and `release` frameworks as output binaries for all macOS, iOS, tvOS, and watchOS targets.
+* Creates a `podspec` task which generates a [Podspec](https://guides.cocoapods.org/syntax/podspec.html)
+file for the project.
 
-The podspec generated includes a path to an output framework and script phases which automate building
-this framework during a build process of an Xcode project. Some fields of the podspec file can be
-configured using the `kotlin.cocoapods { ... }` code block.
+The `Podspec` file includes a path to an output framework and script phases which automate building
+this framework during a build process of an Xcode project. 
 
-<div class="sample" markdown="1" theme="idea" mode="kotlin">
+## Add dependencies on Pod libraries
 
-```kotlin
-// Apply plugins.
-plugins {
-    id("org.jetbrains.kotlin.multiplatform") version "1.3.30"
-    id("org.jetbrains.kotlin.native.cocoapods") version "1.3.30"
-}
-
-// CocoaPods requires the podspec to have a version.
-version = "1.0"
-
-kotlin {
-    cocoapods {
-        // Configure fields required by CocoaPods.
-        summary = "Some description for a Kotlin/Native module"
-        homepage = "Link to a Kotlin/Native module homepage"
-
-        // The name of the produced framework can be changed.
-        // The name of the Gradle project is used here by default.
-        frameworkName = "my_framework"
+You can add dependencies between a Kotlin project and Pod libraries [stored in the CocoaPods repository](#add-a-dependency-on-a-pod-library-from-the-cocoapods-repository) 
+and [stored locally](#add-a-dependency-on-a-pod-library-stored-locally).
+
+[Complete initial configuration](#install-the-cocoapods-dependency-manager-and-plugin), and when you add a new dependency, 
```

Comment:
[Complete the initial configuration](#install-the-cocoapods-dependency-manager-and-plugin), and when you add a new dependency and re-import the project in IntelliJ IDEA, the new dependency will be added automatically. There are no additional steps required.

Reply:
✅